### DISCUSSION TITLE
REQ, detailed ram readout?

### DISCUSSION THREAD

**Discussion by MangyDogUK** [LibreHardwareMonitor/LibreHardwareMonitor#778] [Ideas]
I dont know if there is already a way to do this or if its just not via the normal "for each" method.
But it be good to get more detailed ram data read out.
Such as ram speed, and model and amount in GB rather than %.

**Comment by DerekTurtleRoe** [comment]
This is sort of related:

https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/864

Windows itself can detect some of that information, so I'm guessing adding support for it shouldn't be too difficult? Just an educatd guess. 😅
